doc/rtd/reference/datasources/azure.rst

```diff
@@ -41,6 +41,10 @@ The settings that may be configured are:
   Boolean set to True to use network configuration described by Azure's IMDS
   endpoint instead of fallback network config of DHCP on eth0. Default is
   True.
+* :command:`apply_network_config_for_secondary_ips`
+
+  Boolean set to True to configure primary and secondary IP addresses for each
```

Comment:
```suggestion
  Configure secondary IP addresses for each NIC.
  Default is True.
```

Reply:
Is there a standard template for documenting these options?

Reply:
Unfortunately no. We generally try to keep things consistent, but there's no template or specific rules for it. I'll see if that's something we can add.

Reply:
done!